### ISSUE TITLE
go-chassis will return 406 if `Produces` not been defined but `Accept` exist in request headers

### ISSUE THREAD

**Issue by leikao** [go-chassis/go-chassis#659]
**Describe the bug**
go-chassis will return 406 if `Produces` not been defined but `Accept` exist in request headers.

Test result:
![image](https://user-images.githubusercontent.com/46245845/61168258-de43ef80-a57d-11e9-9b25-95ea125de620.png)

**Version of go chassis**
v1.5.1

**To Reproduce**
1. Register a route to go-chassis, the `Produces` is absent of this route, for example:
>     []Route{
>     	{
>     		Method:           http.MethodGet,
>     		Path:             "/test",
>     		ResourceFuncName: "Handler",
>     	},
>     }
2. Run the application and make a call to the interface defined in step #1, you should add the `Accept` header just like this:
```
curl -H 'Accept: application/json' https://your.endpoint/test'
```
3. You will get a response with status of `406`.

**Logs**
n/a

**Comment by leikao** [comment]
`go-restful` only *accept* requests in case of:
- `Accept` header is absent.
- `Accept` header contains in the `Produces` defined in the route.

there is the logic of `go-restful`:
https://github.com/emicklei/go-restful/blob/a6fb7f2977a2ceeeabf25aeb27eb41f007a95dc6/route.go#L92

**Comment by leikao** [comment]
fixed in [v1.6.0](https://github.com/go-chassis/go-chassis/releases/tag/v1.6.0), close this issue.
